### PR TITLE
Expose Mip Map flags; Fix animation bug regarding bit sets

### PR DESCRIPTION
**Pull request by Eagle517**
I modified the add-on to include UI options for the NoMipMap and MipMapZeroBorder flags as well as making the importer/exporter respect these flags.  Additionally, I made a single change in DtsTypes.py to pack the bit fields as unsigned integers rather than signed integers.  This corrects a bug with complex animations not working properly.

### CODE REVIEW

File: DtsTypes.py [hunk 1]
```diff
@@ -447,7 +447,7 @@ def write_bit_set(fd, bits):
         fd.write(pack("<ii", numWords, numWords))
 
         for word in words:
-                fd.write(pack("<i", word))
+                fd.write(pack("<I", word))
```
Comment (bansheerubber) [inline DtsTypes.py:450]: looks like a good change, a glance at the spec shows that this is the correct data type. however, in `read_bit_set`, it looks like we unpack this data and the types of the words we unpack there should probably be changed from a `int` type to an `unsigned int` type as well.

**Comment by bansheerubber** [comment]
overall LGTM, there's a small change that we should probably make to `read_bit_set` while we're here, but once that's addressed i'll merge into master and release a new version so people can download & install it properly

**Comment by Eagle517** [comment]
Good catch regarding `read_bit_set`.  I originally fixed `write_bit_set` months ago when I ran into the issue while testing complex visibility animations.  Guess I didn't care to check how it imports them.

**Review by bansheerubber — APPROVED** [review]

**Comment by bansheerubber** [comment]
whoops, just found something else that i should've requested in this PR: since mipmapping was disabled by default previously, it should probably stay this way going forward. added that in b255d58a1fa4738646e80ccf69c42dba5ab0fea6 since it was just a one line change

**Pull request merged**